Python/3. operators.md

```diff
@@ -17,14 +17,14 @@ The `%` operator find the reminder of a division.
 
 ### Code
 ```python
-four = 3
-three = 4
+four = 4
+three = 3
```

Comment:
i did that on purpose please revert it back. There has to be some confusion...right? 😆

Reply:
Please no, your confusing everyone

Reply:
lol

Reply:
just saw these, lul. I am adding a few commits right now which will include reverting this as even I am getting confused